test/e2e/specs/wp-signup-spec.js

```diff
@@ -1647,99 +1648,99 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function () {
-				await findADomainComponent.searchForBlogNameAndWaitForResults( blogName );
-				await findADomainComponent.checkAndRetryForFreeBlogAddresses(
-					expectedBlogAddresses,
-					blogName
-				);
-				const actualAddress = await findADomainComponent.freeBlogAddress();
-				assert(
-					expectedBlogAddresses.indexOf( actualAddress ) > -1,
-					`The displayed free blog address: '${ actualAddress }' was not the expected addresses: '${ expectedBlogAddresses }'`
-				);
-				return await findADomainComponent.selectFreeAddress();
-			}
-		);
-
-		step( 'Can see the plans page and pick the free plan', async function () {
-			const pickAPlanPage = await PickAPlanPage.Expect( driver );
-			return await pickAPlanPage.selectFreePlan();
-		} );
-
-		step(
-			'Can then see the sign up processing page which will finish automatically move along',
-			async function () {
-				return await new SignUpStep( driver ).continueAlong( blogName, passwordForTestAccounts );
-			}
-		);
-
-		sharedSteps.canSeeTheOnboardingChecklist();
-
-		step( 'Can see email containing verification link', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-
-			const emailClient = await new EmailClient( signupInboxId );
-			const validator = ( emails ) =>
-				emails.find( ( email ) => email.subject.includes( emailAddress ) );
-			const emails = await emailClient.pollEmailsByRecipient( emailAddress, validator );
-
-			for ( const email of emails ) {
-				if ( email.subject.includes( emailAddress ) ) {
-					return ( verificationLink = email.html.links[ 0 ].href );
-				}
-			}
-			return assert( verificationLink !== undefined, 'Could not locate the login link email.' );
-		} );
-
-		step( 'Can open verification link and verify account', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-			await driver.get( verificationLink );
-			const myHomePage = await MyHomePage.Expect( this.driver );
-			return await myHomePage.isEmailVerified();
-		} );
-
-		after( 'Can delete our newly created account', async function () {
-			// Get username from Account settings page
-			// (it's automatically generated for passwordless signup)
-			const accountSettingsPage = await AccountSettingsPage.Visit( this.driver );
-			const username = await accountSettingsPage.getUsername();
-
-			return await new DeleteAccountFlow( driver ).deleteAccount( username );
-		} );
-	} );
+	// describe.skip( 'Passwordless signup @parallel', function () {
+	// 	const blogName = dataHelper.getNewBlogName();
+	// 	const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
+	// 	const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
+	// 	let verificationLink;
+
+	// 	before( async function () {
+	// 		await driverManager.ensureNotLoggedIn( driver );
+	// 	} );
+
+	// 	step(
+	// 		'Can visit the Jetpack Add New Site page and choose "Create a shiny new WordPress.com site"',
```

Comment:
Can likely be replaced similarly to https://github.com/Automattic/wp-calypso/pull/45500/files#r486311183